### PR TITLE
feat(frames): Frame header and setup frame.

### PR DESCRIPTION
**Pull request by michaelbpaulson**
The frame header and setup frame now have tests.  The tests ensure that
the correct bytes have been produced by the functions within frames/

### CODE REVIEW

**Comment by michaelbpaulson** [comment]
@yunong we can discuss this when I get into the office today.

File: test/frame.spec.js [hunk 1]
```diff
@@ -0,0 +1,36 @@
+'use strict';
+
+var FrameValidationData = require('./FrameValidationData');
+var Frame = require('./../lib/protocol/frame');
+var expect = require('chai').expect;
```
Comment (yunong) [inline test/frame.spec.js]: nitpicky here, but can you please sort these on alphabetical order?

Comment (michaelbpaulson) [inline test/frame.spec.js]: I'll do it write now.

Comment (yunong) [inline test/frame.spec.js]: If you can do it for all of the js files, that would be awesome. I like the
way @donutespresso does it,
https://github.com/Netflix/codex-service/blob/master/lib/handlers/sendPackageBuffers.js#L3-L10
separating different classes of modules and sorting them alphabetically,
similar to C Style.

-Y

**Pull request merged**